kie-server-parent/kie-server-services/kie-server-services-jbpm-ui/src/main/resources/META-INF/services/org.kie.server.services.jbpm.ui.api.UIFormProvider

```diff
@@ -1 +1,4 @@
-org.kie.server.services.jbpm.ui.form.InMemoryFormProvider
\ No newline at end of file
+org.kie.server.services.jbpm.ui.form.RemoteFormModellerFormProvider
+org.kie.server.services.jbpm.ui.form.RemoteKieFormsProvider
+org.kie.server.services.jbpm.ui.form.RemoteInMemoryFormProvider
+org.kie.server.services.jbpm.ui.form.RemoteClassPathFormProvider
```

Comment:
RemoteClassPathFormProvider does not exists any more and that's why build is failing:

Caused by: java.util.ServiceConfigurationError: org.kie.server.services.jbpm.ui.api.UIFormProvider: Provider org.kie.server.services.jbpm.ui.form.RemoteClassPathFormProvider not found